returnn/torch/engine.py

```diff
@@ -64,6 +68,10 @@ def __init__(self, config: Config):
         self._device = _get_device_from_config(config)
         print("Using device:", self._device, file=log.v2)
 
+        self._use_DDP = config.is_true("use_DDP")  # type: Optional[bool]
```

Comment:
As discussed, we want to have this a dict, and rename it, like `torch_distributed`.

Reply:
There would be one `class` or `type` inside that dict, which would be `"DistributedDataParallel"`.